src/3rdparty/walkontable/test/spec/calculator/viewportColumns.spec.js

```diff
@@ -202,4 +202,16 @@ describe('Walkontable.ViewportColumnsCalculator', () => {
     expect(calc.stretchAllColumnsWidth.length).toBe(0);
     expect(calc.needVerifyLastColumnWidth).toBe(true);
   });
+
+  it('should calculate the number of columns based on a default width, ' +
+    'when the width returned from the function is not a number', () => {
+    const calc = new Walkontable.ViewportColumnsCalculator(200, 0, 1000, () => (void 0 + 1));
+    expect(calc.startColumn).toBe(0);
```

Comment:
Can I ask you to add a new line before `const` assignment? I believe that this increases the code readability by encapsulating assignment and logic (expecting) block.

Reply:
This rule should be applied to other lines.

Reply:
Good idea, I'll rearrange other test cases in these files.